docs/en/rules/Azure.Deployment.AdminUsername.md

```diff
@@ -0,0 +1,25 @@
+---
+severity: Awareness
+pillar: Security
+category: Design
+resource: All resources
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.Deployment.AdminUsername/
+---
+
+# Administrator Username Types
+
+## SYNOPSIS
+
+The adminUsername or administratorUsername property should not be a string literal in nested templates.
+
+## DESCRIPTION
+
+In nested deployments a resource can have the adminUsername property set to a string literal value.
+This means that the value is passed through insecurely and is a security risk.
+
+## RECOMMENDATION
+
+Sensitive properties for nested deployments should be set by a secureString parameter.
+
```

Comment:
Insert a `## LINKS` section header. PSRule specifically looks for this, so it must be specifically this. See this as an example: https://github.com/Azure/PSRule.Rules.Azure/blob/main/docs/en/rules/Azure.Storage.SecureTransfer.md

```suggestion

## LINKS

```

In terms of additional links we need to add link to the Azure Well-Architected Framework as a minimum.

Also remove the `/en-us/` from Microsoft links.